server/dba/seeders/prod/05-forestLocations.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    let forestLocations = [
+      { id: 1,  forest_id: 3, district: null, allowed: false, type: 'area', description: 'Wilderness' },
```

Comment:
Have we conducted usability testing on how these are written out? each area is written differently. Let's at least by consistent about whether it says `Road 50` vs `45, 4540` and lets just add an optional `alias` parameter for the human readable names.

Reply:
@lauraGgit Amber gave us the data, but we'll update according to the usability test results.

Reply:
@lpendem please go ahead and make a story to update that.

Reply:
@lauraGgit story created: https://github.com/nciinc/fs-intake-module/issues/80